diff -ruN orig/ecdsa/test_pyecdsa.py new/ecdsa/test_pyecdsa.py
--- orig/ecdsa/test_pyecdsa.py	2019-10-07 07:51:57.000000000 -0600
+++ new/ecdsa/test_pyecdsa.py	2019-11-01 01:11:53.529156505 -0600
@@ -11,7 +11,7 @@
 from binascii import hexlify, unhexlify
 from hashlib import sha1, sha256, sha512
 
-from .six import b, print_, binary_type
+from six import b, print_, binary_type
 from .keys import SigningKey, VerifyingKey
 from .keys import BadSignatureError, MalformedPointError, BadDigestError
 from . import util
@@ -500,10 +500,7 @@
     # vk: 3:OpenSSL->python  4:python->OpenSSL
     # sig: 5:OpenSSL->python 6:python->OpenSSL
 
-    def test_from_openssl_nist192p(self):
-        return self.do_test_from_openssl(NIST192p)
-    def test_from_openssl_nist224p(self):
-        return self.do_test_from_openssl(NIST224p)
+
     def test_from_openssl_nist256p(self):
         return self.do_test_from_openssl(NIST256p)
     def test_from_openssl_nist384p(self):
@@ -539,10 +536,7 @@
         sig = sk.sign(data)
         self.assertTrue(vk.verify(sig, data))
 
-    def test_to_openssl_nist192p(self):
-        self.do_test_to_openssl(NIST192p)
-    def test_to_openssl_nist224p(self):
-        self.do_test_to_openssl(NIST224p)
+
     def test_to_openssl_nist256p(self):
         self.do_test_to_openssl(NIST256p)
     def test_to_openssl_nist384p(self):
